client/css/style.css

```diff
@@ -246,6 +246,15 @@ kbd {
 	color: #7f8c8d;
 }
 
+.session-list strong {
+	display: block;
+}
+
+.session-list p + p {
+	padding-top: 10px;
+	border-top: 1px solid #cdd3da;
```

Comment:
We don't really use lines as separators anywhere else in the app (except under second-level titles) so I think we could do without. But I'm not strongly opinionated either, it's fine with it as well.